tests/phpunit/CRM/Contribute/Form/Task/InvoiceTest.php

```diff
@@ -90,7 +90,7 @@ public function testInvoiceForDueDate() {
     $this->assertContains('PAYMENT ADVICE', $invoiceHTML[$contribution['id']]);
 
     $this->assertContains('AMOUNT DUE: </font></b></td>
-                  <td style = "padding-left:34px;text-align:right;"><b><font size = "1">$ 92.00</font></b></td>', $invoiceHTML[$contribution3['id']]);
+                  <td style="padding-left:34px;text-align:right;"><b><font size="1">$ 92.00</font></b></td>', $invoiceHTML[$contribution3['id']]);
```

Comment:
The test is failing because of the space after "AMOUNT DUE:" (which was cleaned up in the new template).